docs/howto/build-run.md

```diff
@@ -74,6 +74,10 @@ Native upstream on
 You want the tab "React Native CLI Quickstart"; the "Expo CLI Quickstart" does
 not apply.
 
+If you're starting with iOS development, be sure to install CocoaPods,
+as the guide instructs. After you've done that, run
+`cd ios && pod install && cd ..`.
```

Comment:
People working with iOS will also need to run this after each time they run `yarn install` (or plain `yarn`). There's no great place to document that -- really because we should automate it :slightly_smiling_face: as a near-term followup task -- but probably adding it to this paragraph is the best place.

I ran `rg yarn docs/` to scan for other relevant spots. Probably `docs/howto/ios-tips.md` is a good spot to repeat this point.

And then the release instructions is the other spot I noticed... but actually, better than that is `tools/ios`. That already has a line that runs `yarn`; right after that is a good spot to add:
```bash
    ( cd ios && pod install )
```
(The parens mean these commands are run in a ["subshell"](https://www.gnu.org/software/bash/manual/bash.html#Command-Grouping), a new shell that's a child process of the one running the rest of the script. That keeps the `cd` from mutating the main shell, which is slightly cleaner.)

Or oh hey, actually, better:
```bash
    pod install --project-directory=ios
```
Found that from `pod install --help`. The version with two `cd` commands is likely still the right one for these instructions, for their short lifetime before we automate it away, because I think it successfully optimizes for being easy to casually understand.

Reply:
Mmm — it also sounds right, even if we do automate it, to simulate not having successfully run `pod install`, and add a troubleshooting entry for that.

Reply:
Mm, to save you the trouble of looking, I didn't get around to doing this today. But I expect there will be other changes I'll need to make from your review, so I can lump this one in with those. 🙂